bin/cli.js

```diff
@@ -158,6 +158,10 @@ if (opts.slow) {
   mocha.slow(opts.slow)
 }
 
+if (opts.require) {
+  require(fs.absolute(opts.require));
+}
+
```

Comment:
Naming wise either name is fine. Would you expect the documentation (other than the name change and support of multiple files) to remain the same?